### PR TITLE
Filtering false positives of static arguments to wmic /format 

### PR DESCRIPTION
**Pull request by redsand**
See https://petri.com/command-line-wmi-part-3 for example of static formats

### CODE REVIEW

File: rules/windows/process_creation/win_xsl_script_processing.yml [hunk 1]
```diff
@@ -15,10 +15,21 @@ detection:
...
+    - CommandLine|contains: '/Format:List'
+    - CommandLine|contains: '/Format:htable'
+    - CommandLine|contains: '/Format:hform'
+    - CommandLine|contains: '/Format:table'
+    - CommandLine|contains: '/Format:mof'
+    - CommandLine|contains: '/Format:value'
+    - CommandLine|contains: '/Format:rawxml'
+    - CommandLine|contains: '/Format:xml'
+    - CommandLine|contains: '/Format:csv'
+  condition: selection and not false_positives
```
Comment (frack113) [inline rules/windows/process_creation/win_xsl_script_processing.yml:28]: Simply
```yaml
  false_positives:
    CommandLine|contains: 
      - '/Format:List'
      - '/Format:htable'
      - '/Format:hform'
      - '/Format:table'
      - '/Format:mof'
      - '/Format:value'
      - '/Format:rawxml'
      - '/Format:xml'
      - '/Format:csv'
```

**Comment by redsand** [comment]
Hi, is there a way to use regex?  The problem with this, is that using a space or ^ or ` will circumvent the FP detection.

We need something like:
  /Format:[\`\^\s]?+List  or something like that... matching 0 or more of these 3 chars: \s ` and ^

File: rules/windows/process_creation/win_xsl_script_processing.yml (hunk 1 above)
Comment (redsand) [reply rules/windows/process_creation/win_xsl_script_processing.yml:29]: This has been completed, hoping y'all saw my question below.....

**Comment by frack113** [comment]
> Hi, is there a way to use regex?
`|re:`
You Need to test your regex after transformation into the query.
Not all the backend can use regex.

**Comment by redsand** [comment]
Just checking!   Sounds like a lot of sigma detection can be avoided with common command line tricks.   Anyway, unrelated to pr, thanks for answering the question.  PR is ready

**Pull request merged**